demo/screens/SellOrderExecuteScreen.js

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import { connect } from "react-redux";
+import { removeSellOrder } from "../actions";
+import SellOrderExecute from "@presentational/SellOrderExecute";
+import PropTypes from "prop-types";
+import ContractsABIs from "@constants/ContractsABIs";
+const { manaABI, estateABI, marketplaceABI } = ContractsABIs;
+import ContractsAddresses from "@constants/ContractsAddresses";
+const { manaAddress, estateAddress, marketplaceAddress } = ContractsAddresses;
+import { Action } from "tasit-sdk";
+const { Contract } = Action;
+import { createFromPrivateKey } from "tasit-account/dist/testHelpers/helpers";
+
+export class SellOrderExecuteScreen extends React.Component {
+  // TODO: Switch to new DecentralandEstate() once the SDK includes that
+  estateContract = new Contract(estateAddress, estateABI);
+  marketplaceContract = new Contract(marketplaceAddress, marketplaceABI);
+  manaContract = new Contract(manaAddress, manaABI);
+
+  _onOrderExecution = sellOrder => {
```

Comment:
I'm wondering whether we can find a less fintech-like name for `sellOrder`

Reply:
The `Marketplace` contract calls as just `order`, do you prefer?

Reply:
Hmm, I think that's still targets a financial audience. Maybe `LandForSale` / `EstateForSale`?

Reply:
Since the user can buy either `Estates` and `Parcels` I'm not sure but seem that the `Land` is a better generic name.

Reply:
Works for me! It’s still a little confusing because “land” (the contract for land parcels) is land and “estates” are land in English grammar. But I can’t think of a good way around that.

Reply:
Done.

Reply:
Thanks! Resolving this